k8s/sandbox/pub-cert.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
what's this for?

Reply:
It's the public key to create sealed secrets. aat and preview have it as well